### PR TITLE
Add camel-parquet-avro wrapper

### PR DESCRIPTION
**Pull request by f2par0**
camel-parquet-avro wrapper

### CODE REVIEW

**Comment by lopushen** [comment]
reviewing

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -1535,6 +1535,18 @@
         <bundle>wrap:mvn:org.eclipse.paho/org.eclipse.paho.mqttv5.client/1.2.5$overwrite=merge&amp;Export-Package=org.eclipse.paho.mqttv5.*;version=1.2.5</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-paho-mqtt5/${project.version}</bundle>
     </feature>
+    <feature name='camel-parquet-avro' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='[2.16,2.17)'>jackson</feature>
+        <bundle dependency='true'>mvn:commons-io/commons-io/${commons-io-version}</bundle>
+        <bundle dependency='true'>mvn:org.apache.commons/commons-compress/${commons-compress-version}</bundle>
```
Comment (lopushen) [inline features/src/main/feature/camel-features.xml:1542]: not present in the dependency tree, but required by OSGI `avro/1.11.3`

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -1535,6 +1535,18 @@
...
+        <bundle dependency='true'>mvn:org.apache.avro/avro/${avro-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:org.apache.hadoop/hadoop-common/${hadoop3-version}</bundle>
```
Comment (lopushen) [inline features/src/main/feature/camel-features.xml:1544]: not present in the dependency tree, but required by OSGI `camel-parquet-avro/4.5.0.SNAPSHOT`

**Review by lopushen — APPROVED** [review]
no conflict ✔️
compilation works ✔️
no unnecessary usage of wrap protocol ✔️
no unnecessary dependencies ✔️
no hardcoded version if it's in the parent pom ✔️
feature is placed in alphabetical order in the file camel-features.xml ✔️
usage of dependent features (netty, jackson, jakarta ...) ✔️

LGTM

**Review by essobedo — APPROVED** [review]

**Pull request merged**